modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -62,6 +62,75 @@ DataWeave supports these formats (or MIME types) as input and output:
 |===
 // TODO: PLAIN TEXT SECTION? <<format_plain_text>>
 
+[[dw_readers_writers]]
```

Comment:
This dw_readers_writers section was just moved here from another file. It was already been reviewed, quite recently.